include/datetime.php

```diff
@@ -535,16 +536,14 @@ function cal($y = 0,$m = 0, $links = false, $class='') {
 	$o .= '</tr><tr>';
 
 	while ($d <= $l) {
-		if (($dow == $f) && (! $started)) {
-			$started = true;
-		}
+		$started = (($dow == $f) && (!$started));
```

Comment:
This is a difference in functionality. The original code would never set ```$started``` back to "false". This code is able to do so. Is it intended?

Reply:
No, good catch, I went too fast on my formatting spree.